src/lib.rs

```diff
@@ -1073,6 +1081,18 @@ where
         let node_iter = NodeIter::new(table, guard);
         Values { node_iter, guard }
     }
+
+    /// Exntends the [`FlurryHashMap`] with the contents of `iter`.
+    // TODO: Implement this in a not as stupid way
```

Comment:
I think this may actually be the best way to implement it. The exception would be if the `Iterator` implements `ExactSizeIterator`, but that probably requires specialization. I would just remove this comment :)

Maaybe it'd be a good idea to add a `TODO` comment in the body before the loop saying
```rust
// TODO: with specialization, we could eagerly resize if adding the number of elements in an ExactSizeIterator would exceed the capacity of the map.
/// The Java version does this using the tryPresize method.
```

Reply:
Alright, I'll remove the comment then (and add the new one).